lib/determinator/version.rb

```diff
@@ -1,3 +1,3 @@
 module Determinator
-  VERSION = '2.2.1'
+  VERSION = '2.3.0'
```

Comment:
Would you mind doing the version bumps in a separate version bumping PR?

Reply:
Will do - I've amended the PR to have the same version.